service/main.tf

```diff
@@ -0,0 +1,49 @@
+#tfsec:ignore:aws-vpc-no-public-ingress-sgr tfsec:ignore:aws-vpc-no-public-egress-sgr
+module "ecs_alb_service_task" {
+  source  = "cloudposse/ecs-alb-service-task/aws"
```

Comment:
Could we not use 3rd party modules please?  Unless it's from an official provider such as AWS, Hashicorp etc we will need to get security sign off, plus it's an ongoing dependancy / risk that we need to manage. Thanks